music21/test/testStream.py

```diff
@@ -284,6 +285,168 @@ def testBeamsMeasure(self):
         bMeasure.show()
 
 
+class TestMakeBeams(unittest.TestCase):
```

Comment:
These tests need to go within the single "class Test" to run in the music21 testing system.